docs/DeveloperGuide.md

```diff
@@ -1,29 +1,204 @@
+3. The `commandWord` will be used in the switch statement to select the appropriate command. returning `IncorrectCommand` class is the default behavior.
+4. If the `commandWord` is valid, it will run the appropriate `prepareXYZCommand()`.
+5. Each of the individual `prepareXYZCommand()` will take in the userInput without the command portion. The variable is named `userInputNoCommand`. The `prepareXYZCommand()` will check the userInput to see if all the appropriate values are added, and return `XYZCommand` class if the values are correct. `prepareXXXCommand()` will return `IncorrectCommand` class if there are some missing values or inappropriate values.
+
+### Meeting Feature (PeiHao)
+The 'Meeting' Feature allows users to add a meeting, delete a meeting, find a meeting and print all the meetings.
+The 'Meeting' class in the entity package shows the attributes that a meeting object has: time and issue, both are of String type.
+An ArrayList of meetings is initialized in the MeetingManager. Three methods that implement the meetings are also inside.
+- addMeeting: Add a meeting to the meeting list.
+  ![](./uml/AddMeetingCommandSequenceDiagram-Add_Meeting_Sequence_Diagram.png)
+- deleteMeeting: Delete a meeting in the meeting list by its index.
+- printMeetings: Print all the meetings in the meeting list.
+
+When the user input a meeting command, the Parser will determine which command it is and return a command object.
+One of the meeting commands will be called from `AddMeetingCommand`, `DeleteMeetingCommand` and `ViewMeetingCommand`.
+Inside the command, the execute function will call the corresponding methods inside the MeetingManager and implement on the meeting list.
+
+### Deadline Feature (HuiQi)
+The Deadline Feature allows user to add Deadline objects in a deadline list.
+Deadline objects consists of a description String and a dueDate String.
+The User will also be able to delete deadlines, view deadline list.
+
+When the user input a deadline command, it will be parsed into Parser to determine which deadline command will be called.
+Then, the deadline command will be executed, and the respective method in the DeadlineManager class will be called, which are:
+#### addDeadline
+
+   Adds the deadline object to the ArrayList `deadlines`.
+
+   A deadline object can only be added to the deadlines list if it has a valid description and dueDate String.
+
+   If the addition was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been added.
+
+
+####deleteDeadline
+
+   Removes a deadline object from the ArrayList `deadlines` using the index given.
+   A deadline object can only be removed if the index is a valid index in the list.
+   If the deletion was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been deleted.
+
+3. printDeadlines
+   ![](./uml/ViewDeadlineDiagram.png)
```

Comment:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/88618401/228719119-9c0ec6a3-afd4-4f2b-b9a0-7133cb09b183.png">
I think there might be some missing activation bars